lib/eventsource.js

```diff
@@ -15,6 +15,8 @@ var colon = 58
 var space = 32
 var lineFeed = 10
 var carriageReturn = 13
+// 1MB
```

Comment:
nit: could you specify why you landed at 1MB?

Reply:
Good question, TBH I piked up this number without data and expecting that would provide enough head room without impacting to much on the free memory.

Ive just tried different configurations and seems that the sweet spot is 256KBs, beyond this there is no gain in the performance.

```
2KB 9902ms
4KB 8192ms
8KB 5115ms
16KB 2529ms
32KB 1859ms
64KB 911ms
128KB 801ms
256KB 539ms
512KB 562ms
1MB 565ms
```

I will modify the 1MB max threshold to 256KBs

Reply:
Great! Would you mind just adding a comment noting this? You could even link to this if you want to. I just think we should make it clear to someone reading the code how we picked what could be an arbitrary value.

Reply:
done